src/main.py

```diff
@@ -1,258 +1,29 @@
-        dict['Companies'] = companies
-        dict['Job Duration'] = jobs_duration
-        dict['Job Joining Dates']= joining_dates
-        dict['Job Leaving Dates'] = leaving_dates
-
-    num_job.append(len(jobs))
-
-
-def extracting_from_education_section_from_profile(soup,dict,person_no,num_degree):
-    colleges_year_of_grad = []
-    colleges = []
-    degrees = []
-    colleges_year_of_joining = []
-    colleges_duration = []
-
-    for s in soup.find_all('section'):
-            if s.find('div', {'id' : 'education'}) != None:
-                education = s.find('ul')
-
-    school_sections = education.find_all('li', {'class':'artdeco-list__item pvs-list__item--line-separated pvs-list__item--one-column'})
-
-    for school_section in school_sections:
-        years_of_schooling = school_section.find("span", {'class': 't-14 t-normal t-black--light'})
-        year_of_schooling = years_of_schooling.find("span", {'class' : 'visually-hidden'}).get_text().strip()
-
-        college_year_of_join = year_of_schooling.split(' - ')[0]
-
-        if len(college_year_of_join.split()) > 1:
-            college_year_of_join = college_year_of_join.split()[1]
-
-        college_year_of_grad = year_of_schooling.split(' - ')[1]
-
-        if len(college_year_of_grad.split()) > 1:
-            college_year_of_grad = college_year_of_grad.split()[1]
-
-        colleges_year_of_grad.append(college_year_of_grad)
-        colleges_year_of_joining.append(college_year_of_join)
-
-
-        college_duration = int(college_year_of_grad)-int(college_year_of_join)
-        colleges_duration.append(college_duration)
-
-        college_names = school_section.find('span', {'class':'mr1 hoverable-link-text t-bold'})
-        college = college_names.find("span", {'class' : 'visually-hidden'}).get_text().strip()
-        colleges.append(college)
-
-        degree_names = school_section.find('span', {'class':'t-14 t-normal'})
-        if degree_names != None:
-            degree = degree_names.find("span", {'class' : 'visually-hidden'}).get_text().strip()
-            degrees.append(degree)
-        else:
-            degrees.append(' ')
-
-    if person_no > 1:
-        dict['Degree Names'].extend(degrees)
-        dict['School Names'].extend(colleges)
-        dict['School Duration'].extend(colleges_duration)
-        dict['School Joining Dates'].extend(colleges_year_of_joining)
-        dict['School Leaving Dates'].extend(colleges_year_of_grad)
-    else:
-        dict['Degree Names'] = degrees
-        dict['School Names'] = colleges
-        dict['School Duration'] = colleges_duration
-        dict['School Joining Dates'] = colleges_year_of_joining
-        dict['School Leaving Dates'] = colleges_year_of_grad
-
-    num_degree.append(len(degrees))
-
-def gettingDataFromProfiles():
+def getting_data_from_profiles():
 
     driver = webdriver.Chrome()
-    driver.implicitly_wait(5)
 
     iitk_id = 157268
 
     linkedin_urls = []
 
-    logging_in(driver)
```

Comment:
You can break this down into "logging_in" and "getting_profile_links_from_school_id" for better modularization (and more descriptive function naming)